test/blocks/text/text.test.js

```diff
@@ -62,11 +62,20 @@ describe('text block', () => {
       });
     });
   });
-
   describe('two content rows', () => {
     it('has viewport classes', () => {
       const mobileEl = document.querySelector('.text-block .mobile-up');
       expect(mobileEl).to.exist;
     });
   });
+  describe('cta container', () => {
+    it('is added around the last action area', () => {
+      const actionArea = document.querySelector('#has-container');
+      expect(actionArea.parentElement.className.includes('cta-container')).to.be.true;
```

Comment:
Same here. No need for caching
```suggestion
      expect(document.querySelector('#has-container').parentElement.className.includes('cta-container')).to.be.true;
```